dev/vscode-example-configuration/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "type": "shell",
+            "label": "delete-artifacts",
```

Comment:
This seems to be the only command by now. I expected the PR adds a way to run E2E tests. Is this still WIP?